pkg/testutil/setup.go

```diff
@@ -106,19 +106,32 @@ func SetupTestingEnv(params *SetupTestingEnvParams) (logging.Logger, api.ClientW
 
 	s3Endpoint := viper.GetString("s3_endpoint")
 	awsSession := session.Must(session.NewSession())
-	svc := s3.New(awsSession,
+	pathStyleSvc := s3.New(awsSession,
 		aws.NewConfig().
 			WithRegion("us-east-1").
 			WithEndpoint(s3Endpoint).
 			WithDisableSSL(true).
+			WithS3ForcePathStyle(true).
 			WithCredentials(credentials.NewCredentials(
 				&credentials.StaticProvider{
 					Value: credentials.Value{
 						AccessKeyID:     viper.GetString("access_key_id"),
 						SecretAccessKey: viper.GetString("secret_access_key"),
 					}})))
 
-	return logger, client, svc
+	hostStyleSvc := s3.New(awsSession,
+		aws.NewConfig().
+			WithRegion("us-east-1").
+			WithEndpoint(s3Endpoint).
+			WithDisableSSL(true).
+			WithCredentials(credentials.NewCredentials(
+				&credentials.StaticProvider{
+					Value: credentials.Value{
+						AccessKeyID:     viper.GetString("access_key_id"),
+						SecretAccessKey: viper.GetString("secret_access_key"),
+					}})))
```

Comment:
Let's extract common code to a single location and not repeat that when it's not mandatory.